docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -647,6 +647,11 @@ level on the project configuration page or on a build variant level
 within the project. They can be used **as inputs to commands**,
 including shell scripts.
 
+Expansion values defined on the project configurations page that
+    (1) are marked as private OR
+    (2) with key names containing any of the following case-insensitive patterns: `key`, `pass`, `pw`, `token`, `auth`, `private`
```

Comment:
Should we add a blurb about how this shouldn't be relied on and it should be thought of as a last line of defense?

Reply:
Good idea! Done.